static/bidder-info/smartx.yaml

```diff
@@ -0,0 +1,12 @@
+endpoint: "https://bid.smartclip.net/bid/1005"
```

Comment:
endpoint is reachable 

```
~ % curl -i https://bid.smartclip.net/bid/1005
HTTP/2 405
server: nginx/1.17.6
date: Thu, 28 Sep 2023 10:30:07 GMT
content-length: 0
via: 1.1 google
alt-svc: h3=":443"; ma=2592000,h3-29=":443"; ma=2592000
```


@fkoch-sc,  can you provide details on endpoint specifically what does `1005`  corresponds to?

Reply:
1005 is the dedicated endpoint for prebid server in our system. prebid js uses 1000 instead. Helps to keep services separate and tidy.